recipes/klayout/meta.yaml

```diff
@@ -0,0 +1,81 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "klayout" %}
+{% set version = "0.27.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/klayout-{{ version }}.tar.gz
```

Comment:
shouldn't that point to https://github.com/KLayout/klayout/releases ?

Reply:
yep, fixed

Reply:
done